pkg/recommender.py

```diff
@@ -348,76 +363,18 @@ def _schedule_evt(self):
-        morn_min = 0
-        ev_hour = 23
-        ev_min = 0
-
-        # get start time from deployment
         if not self.test_mode:
-            try:
-                #path for DeploymentInformation.db assume recomm system WITHIN acoustic folder
-                depl_info_path = DIR_PATH.replace('\\', '/').replace('caregiver_recomm/pkg','DeploymentInformation.db')
-                #if file doesnt exist revert to testing path
-                depl_info_path = depl_info_path if os.path.isfile(depl_info_path) else \
-                    'C:/Users/Obesity_Project/Desktop/Patient-Caregiver Relationship/Patient-Caregiver-Relationship/DeploymentInformation.db'
-
-                con = None
-                con = sqlite3.connect(depl_info_path)
-                cursorObj = con.cursor()
-
-
-                table_name = 'RESIDENTS_DATA'
-                # select the latest deploymnet by ordering table by created date
-                #must select the second row with 1, 1 because there is both caregivee and caregiver, (time goes in caregiver)
-                cursorObj.execute("SELECT * FROM " + table_name +
-                                " ORDER BY CREATED_DATE DESC LIMIT 1, 1")
-
-
-                # extract start time and end time
-                start_row, end_row = cursorObj.fetchall()[0][11:13]
-                start_hour, start_minute = [int(t) for t in start_row.split(':')]
-                end_hour, end_minute = [int(t) for t in end_row.split(':')]
-
-
-                # For demonstration purposes, morning message sent 1 minute after start, evening message sent 30 minutes before end time
-                # this will be modified later
-                # the following is just for demo purposes:
-                if start_minute == 59:
-                    morn_hour = start_hour + 1
-                    morn_min = 0
-                else:
-                    morn_hour = start_hour
-                    morn_min = start_minute + 1
-                if end_minute >= 30:
-                    ev_hour = end_hour
-                    ev_min = end_minute - 30
-                else:
-                    ev_hour = end_hour - 1
-                    ev_min = 30 + end_minute
-
-            except Exception as e:
-                log('Read SQLite DB error:', e, timer=self.timer)
-            finally:
-                if con:
-                    con.close()
+           self.timer.sleep(180)
 
 
-        # # # # #for testing purposes, remove later (to test evening messages, morning time must be set early)
-        # #self.timer.sleep(10)
-        # morn_hour = 4
-        # morn_min = 25
-        # ev_hour = 13
-        # ev_min = 14
+        # Default message time set as global variables
 
-        TIME_MORN_DELT = timedelta(hours=morn_hour, minutes=morn_min)
-        TIME_EV_DELT = timedelta(hours=ev_hour, minutes=ev_min)
+        # get start time from Informationdeployment.db
+        if not self.test_mode:
+            self.extract_time()
```

Comment:
i think we can move this call to `__init__`

Reply:
Done!